lib/msf/core/module/options.rb

```diff
@@ -67,4 +67,19 @@ def register_options(options, owner = self.class)
     self.options.add_options(options, owner)
     import_defaults(false)
   end
+
+  def register_option_group(name:, description:, option_names: [], merge: true)
+    existing_group = options.groups[name]
+    if merge && existing_group
+      existing_group.description = description
+      existing_group.add_options(option_names)
+    else
+      option_group = Msf::OptionGroup.new(name: name, description: description, option_names: option_names)
+      options.add_group(option_group)
+    end
+  end
+
+  def deregister_option_group(name:)
```

Comment:
Quick question, this method doesn't seem to be called anywhere. At least from what I can see. Just thought I'd double check it's this still required?

Reply:
Cool, was assuming it was just for future proofing 👍